client/src/admin/projects/ExtraDetails.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div v-if="event.id && project.id">
+    <div class="center-grid">
+      <div class="center-grid__content">
+        <div class="error-message-warning">
+          You are an admin editing a live project
+        </div>
+        <extra-details :eventSlug='this.event.slug' :projectId='project.id' :_event='this.event' :_project='project'></extra-details>
```

Comment:
i don' t believe you need "this" in the template

Reply:
Good to know, thanks!